.pre-commit-config.yaml

```diff
@@ -1,23 +0,0 @@
----
-# Copyright (c) Ansible Project
-# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
-# SPDX-License-Identifier: GPL-3.0-or-later
-
-repos:
```

Comment:
@felixfontein You can prevent pre-commit CI from adding commits to PRs using this setting — https://pre-commit.ci/#configuration-autofix_prs. It'd look like this:
```yaml
ci:
  autofix_prs:  false
```
Without this setting, the bot will also be unable to commit to PRs where the authors didn't allow commits from the maintainers.

Reply:
But then it will still show up under checks in CI (and most likely will show "failed", because some tests will fail).

Reply:
Yeah, I didn't anticipate that originally but still felt like this is a useful bit of information to share for the future.